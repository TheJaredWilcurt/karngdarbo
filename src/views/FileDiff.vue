<template>
  <div class="filediff">
    <one-file
      v-for="(file, fileIndex) in files"
      :key="'file' + fileIndex"
      :file="file"
    />
  </div>
</template>

<script>
import OneFile from '@/components/filediff/OneFile.vue';

export default {
  name: 'FileDiff',
  components: {
    'one-file': OneFile
  },
  data: function () {
    return {
      files: [
        {
          path: 'C:\\Folder'
        },
        {
          path: 'C:\\Users\\Bob\\GitHub\\Reponame'
        }
      ]
    };
  }
};
</script>
