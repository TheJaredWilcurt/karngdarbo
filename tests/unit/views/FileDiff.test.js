import { shallowMount } from '@vue/test-utils';

import FileDiff from '@/views/FileDiff.vue';

describe('FileDiff.vue', () => {
  describe('Created', () => {
    test('Default snapshot', () => {
      const wrapper = shallowMount(FileDiff);

      expect(wrapper.html())
        .toMatchSnapshot();
    });
  });
});
