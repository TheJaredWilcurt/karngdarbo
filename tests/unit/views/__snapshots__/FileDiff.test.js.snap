// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FileDiff.vue Created Default snapshot 1`] = `
<div class="filediff">
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
  <one-file-stub file="[object Object]"></one-file-stub>
</div>
`;
